test_files/expected/simplify_xpaths_success_1.txt

```diff
@@ -1,9 +1,9 @@
-pds:logical_identifier
-pds:version_id
-pds:title
-pds:information_model_version
+pds:logical_identifier<1>
```

Comment:
Shouldn't these `<1>` be removed? I think the original expected file was correct.

Reply:
We discussed this on Tuesday's meeting. This change is due to a planned feature.